### DISCUSSION TITLE
Can Vitest UI differentiate between packages?

### DISCUSSION THREAD

**Discussion by iambinlin** [vitest-dev/vitest#3637] [Q&A] [answered]
<img width="728" alt="image" src="https://github.com/vitest-dev/vitest/assets/39019913/ae8b57a4-257f-4ef4-be5c-2fe327f11a06">

I use `vitest` more in the component library to do tests, because the component library is based on `monorepo`, if there is an option to categorize, the test case situation of each package will be clear at a glance, maybe there is such a feature, I just didn't find it, I hope to seek help or answer. 

As follow:

```
- monorepo
   - packageA
     - testFile1
     - testFile2
   - packageB
     - testFile1
     - testFile2
   - ...
```

Thanks a million!

**Comment by sheremet-va** [comment, marked as answer]
Currently UI doesn't support this, feel free to open an issue about this

**Reply by iambinlin** [reply]
Okay, ask if this could be a feature.
